text-ui-test/EXPECTED.TXT

```diff
@@ -1,7 +1,84 @@
-Hello from
- ____        _        
-|  _ \ _   _| | _____ 
-| | | | | | | |/ / _ \
-| |_| | |_| |   <  __/
-|____/ \__,_|_|\_\___|
+   ('-.                 ('-.  ) (`-.               .-')    
```

Comment:
Good job for implementing the text UI testing!

Reply:
Thanks!